src/Nri/Ui/Tabs/V8.elm

```diff
@@ -192,6 +224,33 @@ defaultConfig =
     , spacing = Nothing
     , tabListBackgroundColor = Nothing
     , highContrastTabListBackgroundColor = Nothing
+    , tabListStickyConfig = Nothing
+    }
+
+
+{-| Configure how the top bar is sticky.
```

Comment:
Thanks for the super clear documentation here!

Reply:
I wonder about the mobile attribute -- do we know we need stickiness for mobile at this point? Could we _not_ include this customization yet, and disallow stickiness for mobile?

(There's talk of having tabs collapse down to a different structure entirely for mobile)

Reply:
I just made it configurable because you mentioned it *might* not be a good idea. If we're going to avoid it entirely, I'll just remove the option.

Reply:
Ohh I see. I thought the default meant there was design buy-in on not stickifying for mobile 🤔

Reply:
oh, no, but it'd be good to ask… @bendansby what do you think about having headers in tab components and tables not sticky by default on mobile?

Reply:
and that is how it is in the latest version of this branch, so yay!